packages/medusa/src/api/routes/admin/currency/list-currencies.ts

```diff
@@ -0,0 +1,69 @@
+import { IsOptional, IsString } from "class-validator"
+import { Currency } from "../../../../models"
+import { CurrencyService } from "../../../../services"
+import { ExtendedRequest } from "../../../../types/global"
+import { FindConfig, FindPaginationParams } from "../../../../types/common"
+
+/**
+ * @oas [get] /currencies
+ * operationId: "GetCurrencies"
+ * summary: "List Currency"
+ * description: "Retrieves a list of Currency"
+ * x-authenticated: true
+ * parameters:
+ *   - (query) q {string} Query used for searching currencies.
+ *   - (query) code {string} Code of the currency to search for.
+ *   - (query) includes_tax {boolean} Search for tax inclusive currencies.
+ *   - (query) order {string} to retrieve products in.
+ *   - (query) offset {string} How many products to skip in the result.
+ *   - (query) limit {string} Limit the number of products returned.
```

Comment:
todo: Does not match allowed params. `q`, `code`, and `includes_tax` seems to be missing from `AdminGetCurrenciesParams`